frontend/src/utils/helpers/linkMarkup.js

```diff
@@ -0,0 +1,33 @@
+export const createLinkMarkup = (text, color) => {
```

Comment:
nit: Is the color ever going to change? If not, could we get it from the theme directly in the linkMarkup util?

Reply:
I did look into this.  The color does need to change when you swap from light to dark theme in the settings.  But eslint doesn't like it when the useTheme hook is called within linkMarkup as createLinkMarkup isn't a React component or custom hook.  I looked into wrapping the function in a component, but then calling that started to get a little hacky just for the purpose of avoiding the param.

Reply:
Understood, seems like the solution is more complex than the problem itself. I think we can leave it as is.